README.md

```diff
@@ -30,38 +20,25 @@ different containerized cloud infrastructure deployments.
 
 ## Introduction
 
-Containers are superior to conventional deployment mechanism in that they are:
-
-1. Isolated:
-  - Applications have their own libraries, no conflicts will arise from
-  different libraries in other applications.
-
+Containers are superior to conventional deployment mechanisms in that they are:
 
-2. Limited (limits on CPU/memory):
-  - Applications may not hog resources from other applications.
+1. _Isolated_ - Applications have their own libraries; no conflicts will arise from different libraries in other applications.
 
+1. _Limited (limits on CPU/memory)_ - Applications may not hog resources from other applications.
 
-3. Portable:
-  - Container contains everything it needs, not tied to an OS or Cloud.
+1. _Portable_ - The container contains everything it needs and is not tied to an OS or Cloud provider.
 
+1. _Lightweight_ - The kernel is shared, making it much smaller and faster than a full OS image.
 
-4. Lightweight:
-  - The kernel is shared making it much smaller and faster than a full OS image.
+This project demonstrates migrating a simple Python application named `Prime-flask` to:
 
+1.  A legacy deployment (Debian VM) where `Prime-flask` is deployed as the only application, much like a traditional application is run in an on-premises datacenter
 
-This project demonstrates migrating a simple Python application named Prime-flask:
+1.  A containerized version deployed on [Container-Optimized OS (COS)](https://cloud.google.com/container-optimized-os/)
 
-1.  A legacy deployment (Debian VM) where Prime-flask is deployed as the only
-    application much like a traditional application was run in an
-    on-premises datacenter.
-1.  Second, a containerized version is deployed on Container-Optimized OS (COS).
-1.  Finally, a Kubernetes deployment where Prime-flask runs as a deployment
-    behind a load-balancer cluster in Google Kubernetes Engine.
+1.  A [Kubernetes](https://kubernetes.io/) deployment where `Prime-flask` runs as a deployment behind a load-balancer cluster in Google [Kubernetes Engine](https://cloud.google.com/kubernetes-engine/)
```

Comment:
s/runs as a deployment behind/is exposed via a load balancer and deployed in Kubernetes Engine.

Reply:
k